app/models/register.rb

```diff
@@ -79,6 +79,19 @@ def show_category_link?
     register_phase == 'Beta' && category.present?
   end
 
+  def is_register_published_by_dcms?
+    Authority.where(id: authority_id).first&.government_organisation_key === 'D5'
+  end
+
+  def safe_name
+    # `seo_title` already has 'register' in it. So we need to replace 'register
+    # register' with a 'register'.
+    [[title, seo_title, name].find(&:present?), 'register']
+      .compact
+      .join(' ')
+      .gsub(/register register$/i, 'register')
```

Comment:
I am not sure all the contexts where `seo_title` is used but maybe we can just remove the hardcoded `register` bit from them rather than hacking it in the code. This is OK for now though.

Reply:
Happy to clean up the title / naming spaghetti later - I've raised a [ticket for that](https://trello.com/c/X1LeLGTf/1559-remove-the-uncertainty-around-title-seotitle-and-name).